### DISCUSSION TITLE
Add ability to open statoscope from cli

### DISCUSSION THREAD

**Discussion by Akiyamka** [statoscope/statoscope#38] [Ideas]
Current workflow: 
1) `npm run build:inspect` for generate stats.json
2) find saved link, and open https://statoscope.tech/
3) load stats and inspect

Can be:
1) `npm run build:inspect` generate stats even without save to file, and open https://statoscope.tech/ with this stats

How?
Like what does [termbin.com](termbin.com)
```sh
webpack --profile --json | nc statoscope.tech 9999
```
What do you think?

**Comment by smelukov** [comment]
You can just use `open`-option of Statoscope [webpack plugin](https://github.com/statoscope/statoscope/tree/master/packages/webpack-plugin)

```js
new StatoscopeWebpackPlugin({
  open: 'file'
});
```
